cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -150,6 +150,22 @@ pub(crate) enum Inst {
 
     /// jmpq (reg mem)
     JmpUnknown { target: RM },
+
+    /// (add sub and or xor mul adc? sbb?) (32 64) (reg addr imm) reg
+    SSE_RM_R {
+        is_64: bool, // ?
```

Comment:
is_64 doesn't need to be here; pls rm (also for `SSE_R_R`)

Reply:
I wanted to remove this but kept it around for functions like show_ireg_sized and show_rru_sized. After seeing what it looks like, I don't think it makes sense to create sse variants of basically the same logic. I will simply pass in size=8 for these functions as it seems that may have been the original intent anyway.

Reply:
An XMM reg shouldn't be passed to `show_ireg_sized` (at least, going from memory) -- that's for printing integer register names.  Printing XMM reg names is different in that they are always 16 bytes and there are no names for smaller parts, unlike for the integer registers.  Hence the desire not to have any 'size' qualifiers associated with storage or printing of XMM registers.